templates/cdk-erigon/config.yml

```diff
@@ -625,7 +625,7 @@ sentry.log-peer-info: false
 
 # Drop useless peers, aka peers that return empty body or header responses.
 # Default: false
-sentry.drop-useless-peers: false
+# sentry.drop-useless-peers: false
```

Comment:
Is this flag deprecated now?

Reply:
It seems like it might be. The node wouldn't start with this flag .